recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyjulia" %}
+{% set version = "0.5.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/julia-{{ version }}.tar.gz
+  sha256: 378d0377f75bb0e3bfc4cce19a56d3bf5a9a7be38e370e3a7cf3359bf4cd0378
+
+build:
+  number: 0
+  entry_points:
+    - julia-py = julia.julia_py:main
+    - python-jl = julia.python_jl:main
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv && {{ PYTHON }} -c 'import julia; julia.install()'"
```

Reply:
Since conda uses statically-linked libpython, if the above fails, you would change the last step to: `from julia.api import Julia; Julia(compiled_modules=False); import julia; julia.install()`